test/integration/health_check_integration_test.cc

```diff
@@ -496,7 +496,7 @@ TEST_P(HttpHealthCheckIntegrationTest, SingleEndpointGoAway) {
 // followup health check would happen. Using real time solves this because then
 // the ordering of advancing the time system and enabling the health check timer
 // is inconsequential.
-TEST_P(RealTimeHttpHealthCheckIntegrationTest, SingleEndpointGoAwayErroSingleEndpointGoAwayErrorr) {
+TEST_P(RealTimeHttpHealthCheckIntegrationTest, SingleEndpointGoAwayErroSingleEndpointGoAwayError) {
```

Comment:
Want to fix "Erro" too?

Reply:
Haha. I noticed one typo on the line but not the other. Yes, I'll fix that too.

/wait

Reply:
Kinda makes me wonder if this was just supposed to be `SingleEndpointGoAwayError` and it got pasted in twice with a cursor move in between.